bokeh/models/sources.py

```diff
@@ -283,6 +283,12 @@ class AjaxDataSource(RemoteSource):
     to the server. If this header is supported by the server, then only
     new data since the last request will be returned.
     """)
+    content_type = String(default='application/json', help="""
+    Set the contentType parameter in the ajax call.
+    """)
```

Comment:
Yea, there are standard ones but it can be pretty much be anything.  Its up to the system receiving the request to make use of it.  In the case of Blaze, they've defined their own custom one along the lines of vnd.blaze-json.